runtime/integration-tests/src/pools/utils/time.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+//! Time constants and utlities around time
+
+/// Start date used for timestamps in test-enviornments
+/// Sat Jan 01 2022 00:00:00 GMT+0000
+pub const START_DATE: u64 = 1640995200u64;
+
+pub mod secs {
+	pub const SECOND: u64 = 1000u64;
+	pub const SECONDS_PER_MINUTE: u64 = 60 * SECOND;
```

Comment:
Could these be re-used from https://github.com/centrifuge/centrifuge-chain/blob/parachain/runtime/common/src/lib.rs#L195?

Reply:
Will take a look!

Reply:
I think I keep them although being a duplicate. This makes it more easy to integrate as `time::blocks` and `time::secs`

Reply:
`runtime_common` should _really_ only be used by the three main runtimes. If we want to DRY this up we should move it into `lib/common_something`